lib/src/command_runner.dart

```diff
@@ -77,13 +97,14 @@ class PubCommandRunner extends CommandRunner {
   }
 
   Future run(Iterable<String> arguments) async {
-    var options;
+    ArgResults options;
```

Comment:
Do you need this local anymore? Why not just use `_options` everywhere in the body instead?